templates/submission/status-testcases.html

```diff
@@ -36,7 +36,7 @@ <h3>{{ _('Execution Results') }}</h3>
         <table>{% for case in batch.cases %}
             <tr id="{{ case.id }}" class="case-row toggle closed">
                 <td>
-                    {%- if case.status != 'AC' and case.output -%}
+                    {%- if case.status != 'AC' and case.output and prefix_length > 0 -%}
```

Comment:
Should actually be `prefix_length is None or prefix_length > 0`, since you can't compare `None` with `0`. We also want to be explicit that it can be `None`.